.travis.yml

```diff
@@ -157,20 +157,20 @@ script:
   # - export PYTHONPATH="$PYTHONPATH:./ci/"
 
   # ray tune tests
-  - python python/ray/tune/tests/test_dependency.py
+  - out=`mktemp`; python python/ray/tune/tests/test_dependency.py >$out 2>&1 || cat $out
   # `cluster_tests.py` runs on Jenkins, not Travis.
-  - python -m pytest -v --durations=30 --ignore=python/ray/tune/tests/test_cluster.py python/ray/tune/tests
+  - python -m pytest --durations=10 --ignore=python/ray/tune/tests/test_cluster.py python/ray/tune/tests
 
   # ray rllib tests
-  - python python/ray/rllib/tests/test_catalog.py
-  - python python/ray/rllib/tests/test_filters.py
-  - python python/ray/rllib/tests/test_optimizers.py
-  - python python/ray/rllib/tests/test_evaluators.py
+  - python/ray/rllib/tests/run_silent.sh test_catalog.py
+  - python/ray/rllib/tests/run_silent.sh test_filters.py
+  - python/ray/rllib/tests/run_silent.sh test_optimizers.py
+  - python/ray/rllib/tests/run_silent.sh test_evaluators.py
 
   # ray tests
   # Python3.5+ only. Otherwise we will get `SyntaxError` regardless of how we set the tester.
-  - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest -v --durations=10 python/ray/experimental/test/async_test.py
-  - python -m pytest -v --durations=30 python/ray/tests
+  - python -c 'import sys;exit(sys.version_info>=(3,5))' || python -m pytest --durations=5 python/ray/experimental/test/async_test.py
+  - python -m pytest --durations=10 python/ray/tests
```

Comment:
We may want to keep the `-v`. With it, I can easily find failed test by searching uppercased `FAILED`. Plus, it won't make the log too long.

Reply:
I think this should still be the case. The goal is to suppress output for successful tests. Let me know if you have better ideas on how to do this.

Reply:
AFAIK, the `-v` option only prints the progress with test names. E.g.,
```
test/actor_test.py::test_actor_init_error_propagated PASSED                      [  1%]
test/actor_test.py::test_keyword_args PASSED                                     [  2%]
test/actor_test.py::test_variable_number_of_args PASSED                          [  4%]
test/actor_test.py::test_no_args PASSED                                          [  5%]
test/actor_test.py::test_no_constructor PASSED                                   [  7%]
```

Without `-v`, the progress will be just a series of dots. E.g.,
```
test/actor_test.py .....
```

This doesn't seems to have anything to do with the output.

Reply:
Right, that's the goal. If we print PASSED for each test, that adds thousands of lines of output, which is quite slow to render.

The dots are significantly more compact, and actually makes it easier to search for "failed" tests (actually there is no need to search since the only output is failed test logs).

Reply:
This is going to be an issue for hanging tests, right? When a test hangs and Travis times out, we won't know which test was causing the problem.

Reply:
I believe rendering being slow is due to the build scripts.  Using `-v` should only add a few hundred lines. 
And I think what Robert said is also an issue.